test/parallel/test-stream3-pause-then-read.js

```diff
@@ -35,6 +35,7 @@ let expectEndingData = expectTotalData;
 const r = new Readable({ highWaterMark: 1000 });
 let chunks = totalChunks;
 r._read = function(n) {
+  console.log('_read called', chunks);
```

Comment:
did you mean to add these logs?

Reply:
Yes, they follow the prevailing pattern in this test file. Without the `console.log` statements, this is undebuggable.